tests/src/test/resources/arquillian.xml

```diff
@@ -8,5 +8,6 @@
     <property name="enableImageStreamDetection">false</property>
     <property name="env.init.enabled">true</property>
     <property name="env.dependencies">file://${basedir}/target/test-classes/test-cache.yml</property>
+    <property name="env.config.url">file://${basedir}/target/classes/META-INF/jkube/openshift.yml</property>
```

Comment:
Ideally Arquillian Cube would be able to find this automatically, just like it can find FMP's output. Did you have a chance to look at that? Otherwise very nice work!

Reply:
It's hard-coded there for fabric8 only in several classes.. for example: https://github.com/penehyba/arquillian-cube/blob/476094eaeecd5e1880dd8cc0e32e630624a1bafe/kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/DefaultConfiguration.java#L275

Reply:
You are right :) when globally modified META-INF/fabric8 -> META-INF/jkube it works as expected on OCP4.6 and env.config.url is not necessary https://github.com/penehyba/arquillian-cube/tree/acube2_jkube

On OCP4.5 I use standard ArquillianCube 1.18.2 ..so there are two options, 1st to merge all my drafts (=PRs when all are ready, I want to do it together this week) and ignore OCP4.6 for now or 2nd to wait for OCP4.6 stable/released build and somehow try to include necessary changes of ArquillianCube in jobs... I prefer 1st option, because not everything works on OCP4.6 yet